english/order-pizza/order-pizza.md

```diff
@@ -1,3 +1,10 @@
-Your hungry and tired, pizza is the only thing that will cheer you up.
-But you dont know what to order.
-Suddenly a [menu](https://www.pizzahut.com.au/) appears!
\ No newline at end of file
+Your hungry and tired, pizza is the only thing that will cheer you up. You
+thought long and hard about fried chicken, or perhaps all you can eat mongolian
+stir fry but in the end the only thing that truly matters to a programmer is hot delivery pizza. When it's done you'll hang it from your window in a pillowcase
+to keep it cold and hidden from the horrible people that are after you.
+
+You truly don't know what to order. Pineapple however makes everything better.
+Suddenly a [menu](https://www.pizzahut.com.au/) appears! Another option also
+appears a [menu](http://fongspizza.com/) Fong's appears.
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.